target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -10,30 +10,33 @@ ifeq ($(SUBTARGET),cortexa9)
 
 define Device/linksys
   $(Device/NAND-128K)
-  DEVICE_TITLE := Linksys $(1)
+  DEVICE_VENDOR := Linksys
   DEVICE_PACKAGES := kmod-mwlwifi swconfig wpad-basic
   IMAGES += factory.img
   KERNEL_SIZE := 6144k
 endef
 
 define Device/linksys_wrt1200ac
-  $(call Device/linksys,WRT1200AC (Caiman))
+  $(call Device/linksys)
+  DEVICE_MODEL := WRT1200AC (Caiman)
```

Comment:
Keep the brackets or have an ALT name again? (also below ...)

Reply:
If nobody tells differently, I would keep the brackets version for now.

Reply:
Since nicknames are different for revisions, I changed my mind. I think we should add alternate names, too.

Reply:
```
DEVICE_MODEL := WRT1200AC
DEVICE_ALT0_VENDOR := Linksys
DEVICE_ALT0_MODEL := Caiman
```
That way? Just to be sure.